guide/contributing/designer-guide.md

```diff
@@ -22,23 +22,23 @@ Each design contribution should be supplied in the following viewports:
 
 * XS (mobile): 320 px with container width 288 px
 
-For each viewport supplied, the [TDS grid](http://tds.telus.com/3-Foundational-Elements/4-grid.html) should be applied to ensure consistent sizing and measurements across the system.
+For each viewport supplied, the TDS grid should be applied to ensure consistent sizing and measurements across the system.
 
 #### Page/Layer Naming:
```

Comment:
The heading hierarchy is out of place; I think this should be an h3. Also, please remove the `:` at the end of each heading.